includes/customize-feed.php

```diff
@@ -339,3 +339,24 @@ function empty_rss_excerpt( $output ) {
 }
 // Run it super late after any other filters may have inserted something.
 add_filter( 'the_excerpt_rss', __NAMESPACE__ . '\empty_rss_excerpt', 1000 );
+
+/**
+ * Filter the feed query.
+ * - Default items listed on the feed to 250.
+ *
+ * @param WP_Query $query The WP_Query instance.
+ * @return void
+ */
+function pre_get_posts( $query ) {
+	// do nothing if not the main query.
+	if ( ! $query->is_main_query() ) {
+		return;
+	}
+
+	// Checks run before include current file so is safe to assume we are on the feed query.
```

Comment:
Why don't we use `$query->is_feed()` here?

Reply:
@pabamato see ^

Reply:
@dinhtungdu thanks for the review.
You are correct, I've updated the code to use `$query->is_feed()`.
See: https://github.com/10up/simple-podcasting/pull/109/commits/82194fb6034faafc3adea9fb8d6a4de19249cffc